deployment/terraform/keycloak.go

```diff
@@ -0,0 +1,205 @@
+package terraform
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/assets"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+func (t *Terraform) setupKeycloak(extAgent *ssh.ExtAgent) error {
+	keycloakBinPath := "/opt/keycloak/keycloak-" + t.config.ExternalAuthProviderSettings.KeycloakVersion + "/bin"
+
+	mlog.Info("Configuring keycloak", mlog.String("host", t.output.KeycloakServer.PublicIP))
+
+	command := "start-dev"
+
+	if !t.config.ExternalAuthProviderSettings.DevelopmentMode {
+		command = "start"
+	}
+
+	sshc, err := extAgent.NewClient(t.output.KeycloakServer.PublicIP)
+	if err != nil {
+		return err
+	}
+
+	// Create keycloak.env file
+	var keycloakEnvFileContents []string
```

Comment:
Can we move this declaration closer to where it's used (below the Install realm file if-else block)?